evmlab/tools/statetests/rndval/codesmart.py

```diff
@@ -111,11 +129,27 @@ def create_push_for_data(data):
                     yield create_push_for_data(self.randomSmallMemoryLength())
                     yield create_push_for_data(self.randomSmallMemoryLength())
                     args_filled = True
+                elif instr.name=="CREATE2":
+                    # todo: rework
+                    yield create_push_for_data(self.randomUniInt())
+                    yield create_push_for_data(self.randomSmallMemoryLength())
+                    yield create_push_for_data(self.randomSmallMemoryLength())
+                    yield create_push_for_data(self.randomUniInt())
+                    args_filled = True
+                elif instr.name=="EXTCODEHASH":
+                    # todo: rework
+                    yield create_push_for_data(self.randomSmallUniInt())  # slot
```

Comment:
The item here is an `address`, not a slot, so should be `yield create_push_for_data(RndDestAddress().as_bytes())`, right?